stake-pool/cli/src/main.rs

```diff
@@ -1158,14 +1161,55 @@ fn command_update(
 
     let validator_list = get_validator_list(&config.rpc_client, &stake_pool.validator_list)?;
 
-    let (mut update_list_instructions, final_instructions) =
+    let (mut update_list_instructions, final_instructions) = if force {
```

Comment:
The `--force` flag here is specifically for forcing updates regardless of the last updated epoch. There might be some confusion for CLI users if `--force` suddenly provides more than one override.

Maybe we introduce `--all` or something similar for this?

Reply:
Yeah that would probably be clearer. I wasn't sure if you guys would be ok with adding additional flags at first

Reply:
Yeah, I'd say let's lean on the new flag to isolate the intention!

Reply:
Added the `--fresh` flag that updates all validators when set, while default behavior is to only update the stales ones when the flag is not set